wbooks/src/app/screens/Library/components/Book/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { View } from 'react-native';
+
```

Comment:
delete this empty line

Reply:
this empty line separate library imports from alias imports, not must be separated by empty line?

Reply:
no, we only separate library imports and alias imports from inner imports (inner components, styles, etc.)

Reply:
👍🏼